python/trifinger_simulation/collision_objects.py

```diff
@@ -57,43 +57,40 @@ def import_mesh(
     return obj
 
 
-class Block:
-    """
-    To interact with a block object
-    """
+class Cuboid:
+    """A cuboid which can be interacted with."""
 
     def __init__(
         self,
-        position=[0.15, 0.0, 0.0425],
-        orientation=[0, 0, 0, 1],
-        half_size=0.0325,
-        mass=0.08,
-        **kwargs,
+        position,
+        orientation,
+        half_extends,
+        mass,
+        pybullet_client_id=0,
     ):
         """
         Import the block
 
         Args:
-            position (list): where in xyz space should the block
-                be imported
-            orientation (list): initial orientation quaternion of the block
-            half_size (float): how large should this block be
-            mass (float): how heavy should this block be
+            position (list): Initial xyz-position of the cuboid.
+            orientation (list): Initial orientation quaternion (x, y, z, w) of
+                the cuboid.
+            half_extends (list): Half-extends of the cuboid in x/y/z-direction.
+            mass (float): Mass of the cuboid in kg.
         """
-
-        self._kwargs = kwargs
+        self._pybullet_client_id = pybullet_client_id
 
         self.block_id = pybullet.createCollisionShape(
             shapeType=pybullet.GEOM_BOX,
-            halfExtents=[half_size] * 3,
-            **self._kwargs,
+            halfExtents=half_extends,
```

Comment:
incoherent naming (i guess should be half_extents)

Reply:
fixed